### PR TITLE
feat: add and export ViewMethodOptions

### PR DESCRIPTION
**Pull request by willemneal**

### CODE REVIEW

**Review by chadoh — CHANGES_REQUESTED** [review]
Should be used in and probably exported from `account.ts`, right? https://github.com/AhaLabs/near-api-js/blob/777675f73eef05bdd3746ebb081557e824327d75/src/account.ts#L520

File: src/contract.ts [hunk 1]
```diff
@@ -24,7 +24,6 @@ export interface ChangeMethodOptions {
     walletMeta?: string;
     walletCallbackUrl?: string;
 }
-
```
Comment (chadoh) [inline src/contract.ts]: microscopic nit: we ended up with a missing line here

**Review by chadoh — APPROVED** [review]
Thanks!

File: src/contract.ts [hunk 2]
```diff
@@ -24,7 +24,6 @@ export interface ChangeMethodOptions {
     walletMeta?: string;
     walletCallbackUrl?: string;
 }
```
Comment (willemneal) [inline src/contract.ts]:
```suggestion
}

```

**Pull request merged**